src/app/home/home.module.ts

```diff
@@ -1,3 +1,4 @@
+import { SpaceWizardModule } from '../space-wizard/space-wizard.module';
```

Comment:
As a style standard the imports are organized with @angular first, external imports, then internal imports. There are 2 ways to order them after that both are alphabetically. By the component or the location of the component. For larger imports, I tend to group by the location and then sort by component. For smaller imports I just sort by component only.